ui/Screen/Settings.svelte

```diff
@@ -168,6 +169,20 @@
       </div>
     </section>
 
+    <section>
+      <header>
+        <h3>Peer ID</h3>
+      </header>
+      <div class="section-item">
+        <div class="info">
+          <p>Share your peerId with others so they can add you as a remote</p>
+        </div>
+        <div class="action">
+          <StyledCopyable value={identity.fallback.peerId} />
```

Comment:
What's this `fallback` thing? Should it not just be `identity.peerId`, or is this a frontend thing?

Reply:
Yeah, this is wrong. It should be:
```diff
diff --git a/ui/Screen/Settings.svelte b/ui/Screen/Settings.svelte
index bc57619f..dbb938e8 100644
--- a/ui/Screen/Settings.svelte
+++ b/ui/Screen/Settings.svelte
@@ -1,5 +1,5 @@
 <script>
-  import { onMount } from "svelte";
+  import { getContext, onMount } from "svelte";

   import {
     clear,
@@ -11,7 +11,6 @@
   } from "../src/session.ts";
   import { themeOptions } from "../src/settings.ts";
   import * as path from "../src/path.ts";
-  import * as identity from "../src/identity.ts";
   import * as modal from "../src/modal.ts";
   import { getVersion, isDev } from "../../native/ipc.js";

@@ -22,6 +21,8 @@
     StyledCopyable,
   } from "../DesignSystem/Component";

+  const session = getContext("session");
+
   const updateTheme = event =>
     updateAppearance({ ...$settings.appearance, theme: event.detail });

@@ -178,7 +179,7 @@
           <p>Share your peerId with others so they can add you as a remote</p>
         </div>
         <div class="action">
-          <StyledCopyable value={identity.fallback.peerId} />
+          <StyledCopyable value={session.identity.peerId} />
         </div>
       </div>
     </section>
```

Reply:
:bow: I'll be a frontend dev soon enough

Reply:
<img width="30%" alt="Screenshot 2020-10-30 at 16 26 01" src="https://user-images.githubusercontent.com/158411/97723950-a04d4080-1acc-11eb-99f4-cb2cd2398b3e.png">

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1152/commits/1d6ee2ca9c6df90816d712982a44722adeefae99